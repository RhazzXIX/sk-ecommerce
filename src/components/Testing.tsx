export default function Testing () {
  return (
    <>
      <h1>Hello Up-skilling World!</h1>
      <p>Testing ok</p>
    </>
  )
}