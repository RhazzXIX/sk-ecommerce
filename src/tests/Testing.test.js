import {render} from '@testing-library/react'
import Testing from '../components/Testing'

describe('Trying out testing', ()=> {
  it('Renders on screen', () => {
    const { container } = render(<Testing/>)

    expect(container).toMatchSnapshot()
  })
})