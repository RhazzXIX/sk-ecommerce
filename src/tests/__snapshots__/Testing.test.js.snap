// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trying out testing Renders on screen 1`] = `
<div>
  <h1>
    Hello Up-skilling World!
  </h1>
  <p>
    Testing ok
  </p>
</div>
`;
